packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1216,7 +1216,7 @@ const supportedServices = {
       },
       {
         key: 'signinwithappleTeamIdUserPool',
-        question: 'Enter your Team ID for your OAuth flow: ',
+        question: 'Enter your Apple Team ID for your OAuth flow: ',
```

Comment:
this will require E2E updates as well:
![image](https://user-images.githubusercontent.com/110861985/198719179-9d3ecc69-1ae5-487a-9822-14dfd06f18ac.png)